test/TestProjects/MgmtOperations/Generated/Models/AvailabilitySetGrandChildData.Serialization.cs

```diff
@@ -102,7 +102,7 @@ internal static AvailabilitySetGrandChildData DeserializeAvailabilitySetGrandChi
                     continue;
                 }
             }
-            return new AvailabilitySetGrandChildData(id, name, type, systemData.Value, Optional.ToDictionary(tags), location, bar.Value);
+            return new AvailabilitySetGrandChildData(id, name, type, systemData.Value, Optional.ToDictionary(tags), location, bar);
```

Comment:
This should be expected. The same case in DPG will generate the same code.
And this actually does not change the result - `.Value` is always called when passing to the ctor by the implicit operator